.circleci/config.yml

```diff
@@ -1,241 +1,216 @@
 version: 2.1
 
-slack/notify-on-failure:
-  only_for_branches: "master"
-
+# -------------------------
+#   ORBS
+# -------------------------
 # https://circleci.com/orbs/registry/
 orbs:
   browser-tools: circleci/browser-tools@1.0.1
-  slack: circleci/slack@3.4.2
 
 # -------------------------
-#   PARAMETERS
+#   PIPELINE PARAMETERS
 # -------------------------
 parameters:
-  circleci_service_account:
+  circle-service-account:
```

Comment:
Please add `description` nodes for everything that takes them. This will make it fairly obvious what's left for the README.

I'm really excited to see how clean it's getting: it's going to make expanding and adapting the deployment in the future a breeze.

Reply:
every commands has `description` added.